website/docs/self-type.md

```diff
@@ -19,7 +19,7 @@ we call a method on i.e., `x` in `x.foo`). For instance, `#dup` returns
 type.
 
 ```ruby
-# typed: true
+# typed: strict
```

Comment:
Our docs are not quite consistent, but I think the prevailing style is to use `# typed: true`; please revert this change.

Reply:
```suggestion
# typed: true
```